end_to_end_tests/robot_tests/frontend/series_page_tests/navigation.robot

```diff
@@ -27,29 +27,29 @@ Test Navigation to Last Builds
 Store Series Headers
   Wait Until Element is Enabled   ${series_list}
   ${series}=  Get WebElements   ${series_list}
-  
+
   FOR    ${serial}    IN    @{series}
-      
+
       ${header_text}=    Get Line    ${serial.text}    0
       Append To List    ${series_names}    ${header_text}
   END
 
 Test Overview of All Stored Series
     Wait Until Element is Enabled   ${series_list}
-    
+
     FOR    ${header}    IN    @{series_names}
-      Click Element     //h3[.="${header}"]/ancestor-or-self::div[2]/div[2]/div[1]
+      Click Element     //h4[.="${header}"]/ancestor-or-self::section[2]/div[2]/div[1]
```

Comment:
Ehmm, this selector is not a good idea, we could have a chat about these together.

Reply:
I will change these to function like so
-> The cards will have dynamic ids for different parts based on the unique series name, such as ci_frontend_latest
-> We can then use unique series name as a locator with a the id

Currently What the locator does is it finds the correct header element based on content and then moves one container up from the element after which the element is then found.